src/architecture/utilities/geodeticConversion.cpp

```diff
@@ -16,7 +16,8 @@
  OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
```

Comment:
:large_blue_circle: I point this out not as needing change but simply offering the information; the size of this commit could be reduced by splitting into two. One for the change and then one that adds the tests.

As linked in the CONTRIBUTING.md these two great articles:
https://mtlynch.io/code-review-love/
https://blog.mocoso.co.uk/talks/2015/01/12/telling-stories-through-your-commits/

Reply:
Agreed.  It would be cleaner to do the file update in one commit and then the test in a second.